R/secret.R

```diff
@@ -0,0 +1,89 @@
+# Setup support for the NAME=PASSWORD envvar ----------------------------------
+
+# secret_pw_name("gargle") --> "GARGLE_PASSWORD"
+secret_pw_name <- function(package) {
+  paste0(toupper(package), "_PASSWORD")
+}
+
+# secret_pw_gen() --> "9AkKLa50wf1zHNCnHiQWeFLDoch9MYJHmPNnIVYZgSUt0Emwgi"
+secret_pw_gen <- function() {
+  x <- sample(c(letters, LETTERS, 0:9), 50, replace = TRUE)
```

Comment:
Given you already have sodium pulled in as a dependency, I would be inclined to use `sodium::random` here to avoid the possibility of generating identical passwords as a result of `set.seed`.  It would be a bit of a faff though as you'd need to rejection sample its output, or do something else to map the raw bytes to your pool of characters.